modules/test/playwright/tests/batch-planner/export.spec.ts

```diff
@@ -0,0 +1,408 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataMigrationCenterPagesTest} from '../../fixtures/dataMigrationCenterPages';
+import {exportImportPagesTest} from '../../fixtures/exportImportPages.fixtures';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {streamToJson, unzipFile} from '../../utils/zipFolder';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	exportImportPagesTest,
+	loginTest(),
+	dataMigrationCenterPagesTest,
+	featureFlagsTest({
+		'COMMERCE-8087': true,
+		'LPS-174455': true,
+	})
+);
+
+const stockObjectDefinition = {
+	active: true,
+	externalReferenceCode: 'stockERC',
+	label: {
+		en_US: 'stock',
+	},
+	name: 'Stock',
+	objectFields: [
+		{
+			DBType: 'String',
+			businessType: 'Text',
+			externalReferenceCode: 'nameERC',
+			indexed: true,
+			indexedAsKeyword: true,
+			label: {
+				en_US: 'name',
+			},
+			name: 'name',
+			required: true,
+		},
+	],
+	pluralLabel: {
+		en_US: 'stocks',
+	},
+	portlet: true,
+	scope: 'company',
+	status: {
+		code: 0,
+	},
+};
+
+const stockObjectEntry = {
+	externalReferenceCode: 'nameERC',
+	name: 'Stock Entry',
+};
+
+test('can download jsont format file as json', async ({
+	apiHelpers,
+	dataMigrationCenterPage,
+}) => {
+	const objectDefinition = await apiHelpers.objectAdmin.postObjectDefinition(
+		stockObjectDefinition
+	);
+
+	await apiHelpers.object.postObjectEntry(stockObjectEntry, 'c/stocks');
+
+	await dataMigrationCenterPage.goto();
+	await dataMigrationCenterPage.goToExportFile();
+
+	await dataMigrationCenterPage.exportFile(
+		'JSONT',
+		'C_Stock (v1_0 - Liferay Object REST)',
+		__dirname + '/../../tmp/',
+		['name']
+	);
+
+	const exportPath = path.resolve(__dirname, '../../tmp/Export.zip');
```

Comment:
Can we create a utility function to hide the temp folder? something like `getTempFile('Export.zip'`?